### DISCUSSION TITLE
Let metadata `title` override the project title

### DISCUSSION THREAD

**Discussion by carljoseph** [Zettlr/Zettlr#5151] [Ideas]
I'm used to putting yaml metadata block at the top of my document to include things like the `title`, `author`, etc. It seems as though the `title` parameter is ignored.

It would be nicer (in my opinion anyway), to use the locally prescribed title in the metadata block if it exists, and ignore/override the title in the project properties.

**Comment by nathanlesage** [comment]
But what if you want to use YAML titles to tell your file manager what to show? And what if you have multiple front matters ...?

**Reply by carljoseph** [reply]
Looking [at the manual](https://docs.zettlr.com/en/core/yaml-frontmatter/#frontmatter-variables), shouldn't the yaml metadata drive what pandoc outputs? I could just be doing something wrong here

**Reply by nathanlesage** [reply]
Yes it should but Pandoc will consume all YAML metadata blocks, with the latter overwriting the former. That's why the project title is separate, so that it can be included in the defaults file which in turn has higher priority as the YAML blocks
